src/main/resources/component.mustache

```diff
@@ -1,12 +1,17 @@
 The Snowdrop team is pleased to inform you that we're planning to release a new Snowdrop BOM on *{{releaseDate}}*
 based on Spring Boot *{{version}}*.
 
-Some of our artifacts depend on your product and we need to know which product version we should be using for this new
+Some of our artifacts depend on your component and we need to know which component version we should be using for this new
```

Comment:
Why did you renamed `product` to `component` ? @jacobdotcosta

Reply:
Because in these cases we're managing artifacts only, not a product such as EAP or JWS.

Reply:
Is the word `component` clearly understood by the different stakeholders such as : EAP, ... ?

Reply:
I'm not sure, so I left it as was.

Reply:
AFAIK, JWS is a product like EAP, AMQ, ... and not a component. So I propose to continue part of the description the product word and not component word. WDYT @metacosm ?

Reply:
I have no opinion. It doesn't really matter what we think if people don't understand what we mean. My recommendation would be to not spend 2 weeks on tweaking the text. Just pick a version, create the release ticket, gather feedback from the teams and iterate. 
In this particular instance, I'd put `product/project` if we want to be clearer but that doesn't mean it will be better understood.